assets/js/edd-checkout-global.js

```diff
@@ -355,6 +355,8 @@ function recalculate_taxes(state) {
 		card_zip: $edd_cc_address.find('input[name=card_zip]').val()
 	};
 
+	jQuery('#edd_purchase_submit').after('<span class="edd-loading-ajax edd-loading"></span>');
```

Comment:
Let's move this to display the loading icon after the submit button itself instead of after the fieldset, to avoid style inconsistencies.

So changing `jQuery('#edd_purchase_submit')` to `jQuery('#edd_purchase_submit [type=submit]')`.

This will match the loading icon placement for processing checkout. https://github.com/easydigitaldownloads/easy-digital-downloads/blob/07c3fe173290fa6bb72e7fbd2f06b7b49f5ad126/assets/js/edd-ajax.js#L384 Without this change, any theme adding a padding or margin to the bottom of the fieldset will create space above the loading icon when taxes are being recalculated, but not when the purchase is being processed.